crates/pop-cli/src/commands/test/execute_block.rs

```diff
@@ -192,10 +192,13 @@ mod tests {
 	};
 	use console::style;
 	use pop_common::Profile;
+	use tempfile::tempdir;
 
 	#[tokio::test]
 	async fn execute_block_works() -> anyhow::Result<()> {
-		source_try_runtime_binary(&mut MockCli::new(), &crate::cache()?, true).await?;
+		let cache_path = tempdir().expect("Could create temp dir");
```

Comment:
Better to use temporal file when possible

Reply:
It can simply return a Result with a question mark.

Reply:
Resolved after your comment there https://github.com/r0gue-io/pop-cli/pull/649#discussion_r2392758514